tests/layer_tests/jax_tests/test_select_n.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import jax
+import numpy as np
+import pytest
+from jax import numpy as jnp
+
+from jax_layer_test_class import JaxLayerTest
+
+rng = np.random.default_rng(5402)
+
+
+class TestSelectN(JaxLayerTest):
+    def _prepare_input(self):
+        self.case_num = 2 if (self.input_type == np.bool or self.input_type == bool) else self.case_num
+        cases = []
+        which = rng.uniform(0,self.case_num, self.input_shape).astype(self.input_type)
+        which = np.array(which)
+        for i in range(self.case_num):
+            cases.append(jnp.array(rng.uniform(i*10, (i+1)*10, self.input_shape).astype(self.input_type)))
```

Comment:
check other tests how we initializes floating-point inputs and integer inputs

Reply:
done